### DISCUSSION TITLE
2010/09/01/wpf%E7%95%8C%E9%9D%A2%E7%BC%96%E7%A8%8B%E4%B8%8Elabview%E7%95%8C%E9%9D%A2%E7%BC%96%E7%A8%8B%E7%9A%84%E6%AF%94%E8%BE%83

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#295] [Announcements]
# 2010/09/01/wpf%E7%95%8C%E9%9D%A2%E7%BC%96%E7%A8%8B%E4%B8%8Elabview%E7%95%8C%E9%9D%A2%E7%BC%96%E7%A8%8B%E7%9A%84%E6%AF%94%E8%BE%83

最近一段时间在学习使用WPF编写程序。计算机软件技术发展太快了，新的语言、体系，层出不穷。往往一个技术还没用熟练，就被要求换用更新的技术了。没准哪一天LabVIEW被什么其它语言替代了，我这方面的经验也就全作废了-程序员果然是吃青春饭的。

https://blog.qizhen.xyz/2010/09/01/wpf%E7%95%8C%E9%9D%A2%E7%BC%96%E7%A8%8B%E4%B8%8Elabview%E7%95%8C%E9%9D%A2%E7%BC%96%E7%A8%8B%E7%9A%84%E6%AF%94%E8%BE%83

**Comment by ruanqizhen** [comment]
之前博客下的留言：

hp
2010年09月6日 22:56 编辑
lv程序大了就难搞,要是可以三维连线就好了°

回复
竹贤
2010年09月4日 16:55 编辑
对于某一种特定的语言而言，它的那些专用的开发应用经验肯定会随着这门语言的终结而终结。但就计算机软件工程而言，它的普遍规律和经验，它的发展恐怕就不会是这么短命的吧。宇宙中的一切，包括宇宙本身都是有生有死的。青春决定了一生。从这一点来说，无论你从事何种职业，都是在吃着青春饭。

回复
